examples/kv-events/online/main.go

```diff
@@ -0,0 +1,191 @@
+/*
+Copyright 2025 The llm-d Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+	"syscall"
+
+	"k8s.io/klog/v2"
+
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/kvcache"
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/kvcache/kvblock"
+	"github.com/llm-d/llm-d-kv-cache-manager/pkg/kvcache/kvevents"
+)
+
+const (
+	envHFToken     = "HF_TOKEN"
+	envModelName   = "MODEL_NAME"
+	envZMQEndpoint = "ZMQ_ENDPOINT"
+	envZMQTopic    = "ZMQ_TOPIC"
+
```

Comment:
same question as before - which are really needed by a user? Esp. given this is a controlled sample...

Reply:
This one runs an actual indexer endpoint that is hooked with running vLLMs, so configuration here is useful and is used in the helm chart.